pom.xml

```diff
@@ -4,7 +4,7 @@
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
 
-        <version>1.509.4</version>
+        <version>1.609.1</version>
```

Comment:
1.580.1 would be the usual minimum for Pipeline compatibility (up to 1.4.2). But if the maintainer agrees, this is fine too.